inttest/common/footloosesuite.go

```diff
@@ -201,7 +200,7 @@ func (s *FootlooseSuite) SetupSuite() {
 		// Get a fresh context for the cleanup tasks.
 		ctx, cancel := signalAwareCtx(context.Background())
 		defer cancel(nil)
-		s.cleanupSuite(t, ctx)
```

Comment:
I'd prefer to keep the `t` parameter everywhere.

https://github.com/k0sproject/k0s/issues/2940#issuecomment-1490076989

Reply:
I didn't know the context could change, I changed the order in the parameter.